pshtt.py

```diff
@@ -256,12 +261,17 @@ def hsts_check(endpoint):
 
     # handle multiple HSTS headers, requests comma-separates them
     first_pass = re.split(',\s?', header)[0]
+    second_pass = re.sub('\'', '', first_pass)
```

Comment:
I added a quick and dirty line to strip single quotes out of HSTS headers, which was causing a crash. I think these are actually not valid, raising the priority of tackling https://github.com/dhs-ncats/pshtt/issues/12#issuecomment-241179626.